packages/client/index.js

```diff
@@ -400,7 +400,6 @@ async function plugin (app, opts) {
     client = await buildOpenAPIClient(opts, app.openTelemetry)
   } else if (opts.type === 'graphql') {
     if (!opts.url.endsWith('/graphql')) {
-      /* c8 ignore next 2 */
```

Comment:
Unneeded, as explained on [this comment](https://github.com/platformatic/platformatic/pull/1938/files#r1431121235)